python/MooseDocs/commands/serve.py

```diff
@@ -82,12 +82,10 @@ def _livereload(host, port, config, builder, site_dir):
 def serve(config_file='mkdocs.yml', strict=None, livereload='dirtyreload', clean=True, pages='pages.yml', **kwargs):
     """
     Mimics mkdocs serve command.
-
-    @TODO: When the mkdocs plugin system allows for custom Watcher this should be removed.
     """
 
     # Location of serve site
-    tempdir = os.path.abspath('.moosedocs')
+    tempdir = os.path.abspath(os.path.join(os.getenv('HOME'), '.moosedocs'))
```

Comment:
Because, it your run this with `mkdocs serve`, which it triggers an error that the destination cannot be contained within the source.